sdks/java/core/src/test/java/com/google/cloud/dataflow/sdk/transforms/DoFnWithContextTest.java

```diff
@@ -160,6 +163,14 @@ public void testDoFnWithContextUsingAggregators() {
   }
 
   @Test
+  public void testDefaultPopulateDisplayDataImplementation() {
```

Comment:
Can you add a test here that tests using a `ParDo` of a `DoFnWithContext`? I'm specifically wanting to test this since we (today) take the `DoFnWithContext` and wrap it into a `DoFn` prior to execution. I want to make sure the display data from the underlying `DoFnWithContext` makes it all the way through.